<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtLogin</class>
 <widget class="QWidget" name="QtLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtLogin</string>
  </property>
  <widget class="QPushButton" name="btn_returnstatus">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>290</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>SizeAllCursor</cursorShape>
   </property>
   <property name="text">
    <string>返回状态界面</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_login_send">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_register">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_quit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Edit_name">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>211</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Edit_password">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btn_login_send</sender>
   <signal>clicked()</signal>
   <receiver>QtLogin</receiver>
   <slot>on_btn_login_send_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_register</sender>
   <signal>clicked()</signal>
   <receiver>QtLogin</receiver>
   <slot>on_btn_register_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>75</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_returnstatus</sender>
   <signal>clicked()</signal>
   <receiver>QtLogin</receiver>
   <slot>on_btn_returnstatus_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_quit</sender>
   <signal>clicked()</signal>
   <receiver>QtLogin</receiver>
   <slot>btn_quit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>55</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_btn_login_send_clicked()</slot>
  <slot>on_btn_register_clicked()</slot>
  <slot>on_btn_returnstatus_clicked()</slot>
  <slot>btn_quit()</slot>
 </slots>
</ui>
